packages/design-tokens/scripts/style-dictionary.js

```diff
@@ -0,0 +1,495 @@
+/**
+ * Ported from primer/primitives, as it was removed in v8 release and we still need it
```

Comment:
For posterity, this PR isn't where we'll refactor this file. That will happen in subsequent PRs.